webhook/src/main/java/com/yas/webhook/config/constants/ApiConstant.java

```diff
@@ -0,0 +1,18 @@
+package com.yas.webhook.config.constants;
+
+public final class ApiConstant {
+    public static final String WEBHOOK_URL = "/backoffice/webhooks";
+    public static final String EVENT_URL = "/backoffice/events";
+
+    public static final String CODE_200 = "200";
+    public static final String OK = "Ok";
```

Comment:
You can use org.springframework.http.HttpStatus instead of defining these constants

Reply:
Because the `ApiConstant.CODE_200` is using in the value of annotation. And they don't accept the enum value like `HttpStatus.OK.value()`